src/extension.ts

```diff
@@ -83,6 +95,17 @@ class watermelonPanel {
 		const column = vscode.window.activeTextEditor
 			? vscode.ViewColumn.Beside
 			: undefined;
+		
+		// get commit shas here
+		// exec('git blame -L 1,5 README.md',
+		exec(`git blame -L ${startLine},${endLine} README.md`
```

Comment:
Why `blame`? The author might be out of the company.

Reply:
What's the goal with the blame? I mean, it might be the one, but why?
Github requires the SHA and the owner, not the author.

Reply:
Git blame prints the commit SHAs we need (among other stuff). Once we agree on this approach, we would need some mechanical intuition to wrangle those strings and boom! We got the list of SHAs.

Reply:
Ah, ok great.
I thought for a second to use awk for it but it will fail in non-UNIX systems.
I guess we gotta loop